tests/vm_util_test.py

```diff
@@ -67,5 +67,16 @@ def testReachable_Unreachable(self):
         False, vm_util.IpAddressSubset.REACHABLE, False)
 
 
+class IssueCommandTestCase(unittest.TestCase):
+
+  def testTimeoutNotReached(self):
```

Comment:
Also, will this work on Windows? If not, maybe add a `unittest.skipIf` annotation?